tests/test_profile_page.py

```diff
@@ -0,0 +1,65 @@
+"""Tests related to profile page"""
+import logging
+
+import pytest
+
+from constants import header
+
+
+class TestProfilePage:
+    """Stores tests for profile page base functionality"""
+
+    log = logging.getLogger("[TestProfilePage]")
+
+    @pytest.fixture()
+    def navigate_to_profile_page_via_sign_in(self, start_page, random_login):
+        """
+        - Pre-conditions:
+            - Sign In as a user
+        - Steps:
+            - Click on "My Profile" button
+            - Navigate to profile page
+            - Verify profile username
+        """
+        # Sign In as a user
+        hello_page = start_page.sign_in(random_login)
+
+        # Verify login success
+        hello_page.verify_sign_up_message(username=random_login.username_value)
+
+        # Click on "My Profile" button
+        header.my_profile_button()
+
+        # Navigate to profile page
+        profile_page = hello_page.navigate_to_profile_page()
+        return profile_page
+
+        # Verify profile username
+        profile_page.verify_profile_page_message(username=random_login.username_value)
+
+    def sign_out_from_profile_page_after_sign_in(self, start_page, random_login):
+        """
+        - Pre-conditions:
+            - Sign In as a user
+        - Steps:
+            - Click on "My Profile" button
+            - Navigate to profile page
+            - Click on "Sign Out" button
+            - Navigate to start page
+        """
+        # Sign In as a user
+        hello_page = start_page.sign_in(random_login)
+
+        # Verify login success
+        hello_page.verify_sign_up_message(username=random_login.username_value)
+
+        # Click on "My Profile" button
+        header.my_profile_button()
+
+        # Navigate to profile page
+        profile_page = hello_page.navigate_to_profile_page()
+        return profile_page
```

Comment:
1. You cannot do return from test
2. You cannot return twice

Reply:
Fixed: strings 35 and 61 "return profile_page" was deleted